tests/rbac_test.go

```diff
@@ -0,0 +1,157 @@
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	ktests "kubevirt.io/kubevirt-ansible/tests"
+)
+
+var _ = Describe("Subresource Api", func() {
+
+	flag.Parse()
+
+	var vmi ktests.VMManifest
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		vmi.Name = "vm-cirros"
+		vmi.Manifest = "tests/manifests/vm-cirros.yaml"
+
+		By("Login with the user test_user")
+		args := []string{"login", "-u", "test_admin", "-p", "123456"  }
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.CreateUser()
+		ktests.CreateNamespaces()
+	})
+
+	AfterEach(func ()  {
+		By("Login with the user test_admin")
+		args := []string{"login", "-u", "test_admin", "-p", "123456"  }
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Delete identity of user kubevirt-test-user")
+		args = []string{"delete", "identity", "allow_all_auth:kubevirt-test-user" }
+		_, err = tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.DeleteUser()
+		ktests.RemoveNamespaces()
+	})
+
+	Describe("Rbac Authorization", func() {
+
+		It("with admin permission", func() {
+
+				By("Add admin role to user kubevirt-test-user")
+				args := []string{"adm", "policy", "add-role-to-user", "admin", ktests.UsernameTestUser, "-n", tests.NamespaceTestDefault }
+				_, err := tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Login with the user kubevirt-test-user")
+				args = []string{"login", "-u", ktests.UsernameTestUser, "-p", "123456"  }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Create a VM with yaml")
+				args = []string{"create", "-f", vmi.Manifest, "-n", tests.NamespaceTestDefault }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check VMI is created")
+				args = []string{"start", vmi.Name, "-n", tests.NamespaceTestDefault }
+				_, err = tests.RunCommand("virtctl", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the vmi is running")
+				args = []string{"get", "vmi", vmi.Name, "--template", "{{.status.phase}}", "-n", tests.NamespaceTestDefault }
+				Eventually(func() string {
+					output, err := tests.RunCommand("oc", args...)
+					Expect(err).ToNot(HaveOccurred())
+					return output
+				}, 50).Should(Equal("Running"))
+
+				By("Connect via console")
+				vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+				expecter, err := tests.LoggedInCirrosExpecter(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				defer expecter.Close()
+
+		})
+
+		It("with edit permission", func ()  {
+
+				By("Add admin role to user kubevirt-test-user")
+				args := []string{"adm", "policy", "add-role-to-user", "edit", ktests.UsernameTestUser, "-n", tests.NamespaceTestDefault }
```

Comment:
These 2 tests with `admin` and `edit permission seems quite similar with just differing in which permission to set for the user, so this should be put in Entry table